packages/core/src/components/discord-embed/DiscordEmbed.ts

```diff
@@ -388,18 +398,35 @@ export class DiscordEmbed extends LitElement implements DiscordEmbedProps, Light
 	private parseTitle(title?: string) {
 		if (!title) return null;
 
-		const words = title.split(' ');
+		const el: (TemplateResult<1> | string)[] = [];
+		let complete = '';
+
+		for (const words of title.split('\n')) {
+			for (const word of words.split(' ')) {
+				const emoji = getGlobalEmojiUrl(word) ?? this.embedEmojisMap[word] ?? ({} as Emoji);
+
+				if (emoji.name) {
+					el.push(html`<discord-custom-emoji name=${emoji.name} url=${ifDefined(emoji.url)} embed-emoji></discord-custom-emoji>`);
+				} else {
+					complete += `${word} `;
+				}
+
+				if (complete === ' ') {
+					el.push(html`<br />`);
+				}
+			}
+
+			el.push(complete);
+
+			complete = '';
+		}
 
-		return words.map((word: string, idx: number) => {
-			const emoji = getGlobalEmojiUrl(word) ?? this.embedEmojisMap[word] ?? ({} as Emoji);
-			let el;
-			if (emoji.name) {
-				el = html`<discord-custom-emoji name=${emoji.name} url=${ifDefined(emoji.url)} embed-emoji></discord-custom-emoji>`;
-			} else {
-				el = idx < words.length - 1 ? `${word} ` : word;
+		return el.map((wordOrHtmlTemplate) => {
+			if (typeof wordOrHtmlTemplate === 'string') {
+				return html`<span>${wordOrHtmlTemplate}</span>`;
 			}
 
-			return el;
+			return wordOrHtmlTemplate;
 		});
```

Comment:
In your code you typed `word` (now `wordOrHtmlTemplate`) as `string`, possibly because it was typed as any above. This hid a bug however because in one of the branches it is not a string but an HTML TemplateResult with `discord-custom-emoji` and in that case it should not be wrapped with another TemplateResult holding a `span`. By strictly typing `el` this is revealed more clearly and you can add a `typeof` check to ensure that only raw strings get wrapped with a span.